src/qtgui/plotter.cpp

```diff
@@ -2238,22 +2248,23 @@ void CPlotter::drawOverlay()
     }
 
     // draw amplitude values (y axis)
-    for (int i = 0; i < m_VerDivs; i++)
+    for (int i = 0; i <= m_VerDivs; i++)
     {
         qreal y = h - ((double)i * pixperdiv + adjoffset);
         qreal th = metrics.height();
+        qreal th_2 = th / 2;
```

Comment:
Minor: adding another variable doesn't really make things clearer or improve performance.

Reply:
I was reusing the same value three times, so I felt having it as a variable was cleaner, but that’s a matter of opinion.

Reply:
I can remove the extra variable and just to th/2 in the added comparison

Reply:
Changed it back to th / 2, I decided the change is cleaner without introducing an extra variable.